vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts

```diff
@@ -38,13 +38,22 @@ export class PageViewPerformanceManager {
          *  |---network---||---request---|---response---|---dom---|
          *  |--------------------------total----------------------|
          */
+        var navigationTiming = this.getPerformanceNavigationTiming();
```

Comment:
Is this value more accurate or why is being added now?

Reply:
`PerformanceTiming` interface is deprecated : 
https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming
So add `PerformanceNavigationTiming` interface for supported browsers: 
https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming